js/types.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+export type NetInfoType =
+  // iOS & Android
+  | 'none'
+  | 'cellular'
+  | 'unknown'
+  | 'wifi'
+  // Android only
+  | 'bluetooth'
+  | 'ethernet'
+  | 'wimax';
+
+export type NetInfoEffectiveType = 'unknown' | '2g' | '3g' | '4g';
```

Comment:
is it to soon to put a 5g in there now?

Reply:
Neither platform currently supports 5g network types yet. I'll add it in later incase they decide to rebrand it as something else.